### ISSUE TITLE
 I can't get a good result

### ISSUE THREAD

**Issue by mengluffy** [sergivalverde/cnn-ms-lesion-segmentation#5]
Dear professor, when I use CHB and UNC data training , I can't get a good result .
about data pre-processing , I use BET skull-stripped ,but no N3 intensity normalized and smoothing ,also no resampled to 256*256*256 , then i train it , so because pre-processing and other?
Can you give me a suggestion.

**Comment by sergivalverde** [comment]
hi Mengluffy

N3 and smoothing should not affect segmentation if training and testing images are preprocessed equally.  

The main problem with the original dimensions (512x512x512) is that it takes a lot to train. However we had similar results that when we resampled masks before training. I encourage you to downsample masks in order to speed up training. 

Take into account that you have 2 raters. What we did here was to train always against the CHB rater. In our experiments, we ended up discarding the UNC rater because was somehow undersegmenting lesions. So, when testing on the UNC rater we had a slightly lower accuracy. 

Can you provide me more information on your training setup please?

**Comment by mengluffy** [comment]
Thank you very much.
My computer configuration is CPU i7 6700, GPU quadro M2000, RAM 16GB. It takes about six to eight hours to train
I use UNC data (train_part 1 case 1 and case 2, test case 3 and case 4)
because my computer RAM is 16 GB, I set epoch=100, batch_size=2000,

**Comment by sergivalverde** [comment]
So you train with just two images from scratch? If this is the case then
you probably overfit the network. In our experiments we were training with
a leave one out approach (19 images for training and one testing).

I would increase the number of training cases.

**Comment by mengluffy** [comment]
Thanks, I try increase the number of training cases.
And I don't know what "mask" is in this paper or code.
(it is a image? or  image mask operation  )
